README.md

```diff
@@ -203,6 +212,9 @@ To remove Dapr from your Kubernetes cluster, use the `uninstall` command with `-
 
 ```
 $ dapr uninstall --kubernetes
+
+# Uninstall v0.1.0 runtime
```

Comment:
First list the uninstall command for taking the specific version and then you can give an example..